telemetry/specs/INT.mdk

```diff
@@ -1380,7 +1452,7 @@ control Int_egress(inout headers hdr, inout metadata meta,
 
     apply{
         if(hdr.int_header.isValid()) {
-            if(hdr.int_header.max_hop_cnt != hdr.int_header.total_hop_cnt 
+            if(hdr.int_header.max_hop_cnt != hdr.int_header.total_hop_cnt
```

Comment:
We can add a condition that checks MTU limit and an action that sets the M bit.

Reply:
Int_egress currently does not have access to egress MTU. Can you please modify the P4 program to pass in egress MTU to Int_egress and set the M bit if needed in Int_egress?